circle.yml

```diff
@@ -15,17 +15,17 @@ test:
         files:
           - client/**/Makefile
           - server/**/Makefile
-    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-client.xml npm run test-client -- --reporter=mocha-junit-reporter -w:
+    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-client.xml npm run test-client -- --reporter=mocha-junit-reporter:
         parallel: true
         files:
           - client/**/test/*.js
           - client/**/test/*.jsx
-    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-server.xml npm run test-server -- --reporter=mocha-junit-reporter -w:
+    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-server.xml npm run test-server -- --reporter=mocha-junit-reporter:
         parallel: true
         files:
           - server/**/test/*.js
           - server/**/test/*.jsx
-    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-tests.xml npm run test-tests -- --reporter=mocha-junit-reporter -w:
+    - NODE_ENV=test make client/config/index.js && MOCHA_FILE=./test-results-test.xml npm run test-test -- --reporter=mocha-junit-reporter:
```

Comment:
can we make the building of the config an earlier part of the run and stop repeating it? 

I feel like I tried it and circle did some weird things.. Might be worth trying again to DRY this up.